api/v1beta2/runtimecomponent_types.go

```diff
@@ -325,6 +325,7 @@ type RuntimeComponentStatus struct {
 
 	// +operator-sdk:csv:customresourcedefinitions:type=status,displayName="Status Conditions",xDescriptors="urn:alm:descriptor:io.kubernetes.conditions"
 	Conditions     []StatusCondition `json:"conditions,omitempty"`
+	Endpoint       string            `json:"endpoint,omitempty"`
```

Comment:
It'll be good to have a type definition suggested [here](https://playbook.cloudpaklab.ibm.com/operator-custom-resource-status/#Endpoints). Let's not worry about caSecret field for now, we can add that later on via a separate PR (it's related to what Artur is working on). Values for `type` and `name` can be fixed as below. The `name` field can be used to identify/lookup an endpoint. In the future, we can also add the Service endpoint as an Internal endpoint. 

```
status:
  endpoints:
    - type: Application
      name: external-route
      scope: External
      uri: https://iaf-system-ui.acme-iaf.acme.com
```